mbuild/coordinate_transform.py

```diff
@@ -670,7 +670,7 @@ def x_axis_transform(compound, new_origin=None,
         new_origin = np.asarray(new_origin)
     else:
         raise TypeError('x_axis_transform, y_axis_transform, and z_axis_transform only accept'
-                        ' np.ndarrays, mb.Compounds, lists, or None of size 3 for the new_origin'
+                        ' np.ndarrays, mb.Compounds, lists, tuple, or None of size 3 for the new_origin'
```

Comment:
We tend to just lump descriptions like this into "list-like" so I'd probably write it as 

`mb.Compounds, list-like of length 3, or None for the new_origin'